<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocketServer</class>
 <widget class="QDialog" name="SocketServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SocketServer</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>47</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>PortNo:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePortNo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1234</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Protocol</string>
   </property>
   <widget class="QRadioButton" name="rbTCP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>83</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TCP</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbUDP">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>83</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>UDP</string>
    </property>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="teLog">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>401</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbStartService">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>StartService</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbStopService">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>StopService</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbExit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leStatus">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>[Connection Table]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>47</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>[Log]</string>
   </property>
  </widget>
  <widget class="QTableView" name="tvCntTbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>401</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lePortNo</tabstop>
  <tabstop>rbTCP</tabstop>
  <tabstop>rbUDP</tabstop>
  <tabstop>teLog</tabstop>
  <tabstop>pbStartService</tabstop>
  <tabstop>pbStopService</tabstop>
  <tabstop>pbExit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
